frappe/website/website_theme/standard/standard.json

```diff
@@ -8,13 +8,12 @@
  "doctype": "Website Theme",
  "font_properties": "300,600",
  "idx": 27,
- "modified": "2020-04-19 05:34:56.086984",
+ "modified": "2020-04-20 23:42:31.649416",
  "modified_by": "Administrator",
  "module": "Website",
  "name": "Standard",
  "owner": "Administrator",
- "primary_color": "Blue",
```

Comment:
@Thunderbottom this was the problem.

The standard theme is rebuilt after deply so the `theme_url` is harmless.